java/ql/lib/ext/com.google.common.io.model.yml

```diff
@@ -3,6 +3,13 @@ extensions:
       pack: codeql/java-all
       extensible: sinkModel
     data:
+      - ["com.google.common.io", "Files", False, "asCharSink", "(File,Charset,FileWriteMode[])", "", "Argument[0]", "write-file", "ai-generated"] # this is a bit like a stream
```

Comment:
Usual counter-intuitive gotcha: `write-file` refers to the contents being written. So this is actually `create-file`:

```suggestion
      - ["com.google.common.io", "Files", False, "asCharSink", "(File,Charset,FileWriteMode[])", "", "Argument[0]", "create-file", "ai-generated"]
```